sdk/openai/openai/review/openai-rest.api.md

```diff
@@ -1223,6 +1227,14 @@ export type OpenAIContext = Client & {
     path: Routes;
 };
 
+// @public
+export interface OpenAIErrorModel {
```

Comment:
Could we give this the good name and use a different name for the class?
```suggestion
export interface OpenAIError {
```

Reply:
Never mind, resolving it.

Reply:
I will change the error class name into `OpenAIErrorClass` and the interface name to `OpenAIError`